pages/live.tsx

```diff
@@ -0,0 +1,203 @@
+import { Container } from "../components/util/container";
+import { SEO } from "../components/util/seo";
+import { VideoCard } from "../components/util/videoCards";
+import { getEvents, getSpeakersInfoFromEvent } from "../services/server/events";
+import { getYoutubePlaylist } from "../services/server/youtube";
+
+export default function LivePage(
+  props: InferGetStaticPropsType<typeof getStaticProps>
+) {
+  const { data } = useTina({
+    data: props.data,
+    query: props.query,
+    variables: props.variables,
+  });
+  const [visibleVideos, setVisibleVideos] = useState(9);
+
+  return (
+    <Layout>
+      <SEO seo={data.live.seo} />
+      <LiveHeader {...data.live} />
+      <Container size="xsmall">
+        <span className="text-sswRed">
+          <h2>{data.live.section1}</h2>
+        </span>
+        <div>
+          {props.event?.Title && (
+            <div className="col-span-2">
+              <div className="whitespace-pre-wrap text-2xl font-bold">
+                {props.event?.Title}
+              </div>
+            </div>
+          )}
+        </div>
+        <div>
+          {props.speaker && (
+            <>
+              <div className="pb-3 text-lg">
+                With{" "}
+                <PresenterLink
+                  name={props.speaker?.Title}
+                  profileLink={props.speaker?.PresenterProfileLink}
+                />
+              </div>
+              <br />
+            </>
+          )}
+        </div>
+        <div className="grid grid-cols-1 justify-center gap-8 lg:grid-cols-2">
+          <div>
+            {props.event?.Abstract && (
+              <div className="col-span-2">
+                <div className="whitespace-pre-wrap text-lg">
+                  {props.event?.Abstract}
+                </div>
+              </div>
+            )}
+          </div>
+          <div className="float-left">
+            {props.event?.Url &&
+              props.event?.Title &&
+              (props.event?.TrailerUrl?.Url ? (
+                <VideoCard
+                  link={props.event.TrailerUrl.Url}
+                  title={props.event.Title}
+                  theme="light"
+                />
+              ) : (
+                <VideoEmbed
+                  data={{
+                    url: "https://www.youtube.com/watch?v=coPZ75akNYA",
+                    videoWidth: "w-full",
+                    removeMargin: true,
+                  }}
+                />
+              ))}
+          </div>
+        </div>
+        <div className="flex justify-center">
+          <UtilityButton
+            size="small"
+            uncentered={false}
+            link="https://www.youtube.com/channel/UCBFgwtV9lIIhvoNh0xoQ7Pg"
+            buttonText={
+              <>
+                Visit SSW TV Channel on Youtube
+                <BsArrowRightCircle className="ml-1 inline" />
+              </>
+            }
+            noAnimate
+            isTargetBlank={true}
+          />
```

Comment:
Make link and the button text Tina fields.

Reply:
Done